exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,12 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var percentageOfStudent = Math.round(
```

Comment:
You can create a variable which can be called 'total' to add both 'numberOfStudents' and 'numberOfMentors' together, this will make the code easy to read.